miso-web/src/main/webapp/scripts/hot_boxable.js

```diff
@@ -69,19 +69,21 @@ HotTarget.boxable = (function() {
     for ( var key in freeByAlias) {
       freeByAlias[key].sort(sortFunction);
     }
+    var changes = [];
     for (var row = 0; row < rowCount; row++) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "row" as this name is already used in declaration at line 57. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2814)